Backend/Remora.Discord.Rest/Polly/DiscordRateLimitPolicy.cs

```diff
@@ -162,6 +163,9 @@ bool continueOnCapturedContext
             return response;
         }
 
+        // TODO does moving this cacheOptions up here cause regressions?
```

Comment:
Does it? :P

Reply:
On second pass, no reason why it should.